src/AdventOfCode.jl

```diff
@@ -21,13 +21,16 @@ function _download_data(year, day)
     error("Unable to download data")
 end
 
-function _template(year, day)
-    data_path = normpath(pwd() * "data/$year/day_$day.txt")
+function _template(year, day; include_year = true)
+    rel_path = "data/day_$day.txt"
+    include_year && (rel_path = joinpath(string(year), rel_path))
+    
+    data_path = normpath(pwd() * rel_path) # this isn't being used?
     """
     # $(_base_url(year, day))
     using AdventOfCode
 
-    input = readlines("data/$year/day_$day.txt")
+    input = readlines("$rel_path")
```

Comment:
we can probably skip the string interpolation here:
```suggestion
    input = readlines(rel_path)
```

Reply:
I tried that first actually, but that gets expanded to eg `readlines(2020/data/day2.txt)`, which doesn't parse 🙃

Reply:
Ah that's right, I missed that this was inside the triple-quoted block. The current code looks good to me now